zebra-state/src/constants.rs

```diff
@@ -13,3 +15,11 @@ pub const MAX_BLOCK_REORG_HEIGHT: u32 = MIN_TRANSPARENT_COINBASE_MATURITY - 1;
 
 /// The database format version, incremented each time the database format changes.
 pub const DATABASE_FORMAT_VERSION: u32 = 4;
+
+use lazy_static::lazy_static;
+use regex::Regex;
+
+lazy_static! {
+    /// Regex that matches the RocksDB error when its lock file is already open.
+    pub static ref LOCK_FILE_ERROR: Regex = Regex::new("(lock file).*(temporarily unavailable)|(in use)").expect("regex is valid");
```

Comment:
Add Windows error from:
https://github.com/ZcashFoundation/zebra/pull/1535/checks?check_run_id=1788192685#step:7:1700
```suggestion
    pub static ref LOCK_FILE_ERROR: Regex = Regex::new("(lock file).*(temporarily unavailable)|(in use)|(being used by another process)").expect("regex is valid");
```